website/src/pages/accelerate-dev-onboarding.tsx

```diff
@@ -0,0 +1,102 @@
+                        </h4>
+                        <p>
+                            With Sourcegraph, developers can find their own answers without waiting for someone to point
+                            them to the relevant code. Get sharable links to help new developers ask specific questions
+                            with context included, and enable senior devs to share their knowledge without the friction
+                            of running local searches for specific bits of code.
+                        </p>
+
+                        <div className="mt-5">
+                            <div id="form-0" />
+                        </div>
+                    </div>
+
+                    <div className="col-lg-6 mt-4 mt-lg-0">
+                        <div className="embed-responsive embed-responsive-16by9">
+                            <iframe
+                                src="https://www.youtube.com/embed/DgwvhRW1Cbc"
+                                title="Accelerate developer onboarding with Sourcegraph"
+                                allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+                                allowFullScreen
+                                className="embed-responsive-item"
+                            />
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+            <div className="bg-gradient-onahau-fog text-center">
+                <div className="container col-lg-6 py-6">
+                    <h2 className="mb-4 font-weight-bold">
+                        See how Convoy increases the efficiency and confidence of entry level developers
+                    </h2>
+                    <blockquote>
+                        <p>
+                            &ldquo;For our new developers, Sourcegraph has been invaluable to get to know the repository
+                            structure, to track down where code lives, and self-service during their
+                            investigations.&rdquo;
+                        </p>
+                        <figcaption className="text-muted mt-4">
+                            —Owen Kim, Senior Software Engineer at Convoy
+                        </figcaption>
+                    </blockquote>
+
+                    <img src="/external-logos/convoy-logo.svg" alt="Convoy logo" width={130} className="mt-4" />
+
+                    <Link to="/case-studies/convoy-improved-on-boarding" className="d-block mt-5 font-weight-bold">
+                        Read the full case study
+                    </Link>
+                </div>
+            </div>
+
+            <div className="text-center">
+                <div className="container col-lg-8 py-8">
+                    <h2 className="font-weight-bold">
+                        Time to think about your developer onboarding workflow? Let's talk!
```

Comment:
Does this text need to match the Figma? Not sure if this is the most up-to-date Figma — it's the one linked from the issue.
![Screen Shot 2022-03-15 at 9 59 41 AM](https://user-images.githubusercontent.com/43454804/158401218-4808b35a-0d0e-4206-aa90-5eefe8902cc8.png)

Reply:
I believe they're supposed to match the copy doc as the source of truth and the Figma is a visual outline. So I would say I don't think so. But just to double check – @marybelzer is that correct?

Reply:
Figma is the source of truth, so we should make sure the page matches Figma.

Reply:
Thanks for confirming @marybelzer! I'll push up some updates with the changes.